src/routes/book.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const bookController = require('../controllers/book');
+
+const router = express.Router();
+
+router.post('', bookController.addBook);
+router.get('', bookController.getAllBooks);
```

Comment:
Although there is nothing wrong with this, it is considered best practice to put a / between the quotations in the first parameter if there is no specific route after the base route
```
router.post('/', bookController.addBook);
router.get('/', bookController.getAllBooks);
```